### PR TITLE
bug fix for setting up remote route from linux

### PR DESCRIPTION
**Pull request by chrisbeardy**
attempting to resolve issue #66 (or dereiative of) 

The simple solution appears to be to swap hostname and route_name, this works for TWINCAT 3.1.3024. and 3022. However I don't have TWINCAT 2 or 3.1.3020 so not sure how this will affect that. 

@Adrian-at-CrimsonAuzre I'm not sure your thoughts on this? What version of TwinCat did you implement in? 

possibly Beckhoff they have switched these bytes around?

Does anyone know how we would tell what version of TwinCat is running?

### CODE REVIEW

**Comment by chrisbeardy** [comment]
thanks for such a quick response @Adrian-at-CrimsonAuzre, I shall maybe at some point have a look at #75 to see if I can get something to work for both TC2 and TC3 but I'm the other way to you, I've not access to TC2 ha

**Comment by chrisbeardy** [comment]
Sorry for my absence, just changed jobs...however that has given me access to TC2 and I can confirm that this fix works for me on TC2, TC3 4022 and 4024.

File: pyads/ads.py [hunk 1]
```diff
@@ -361,15 +361,15 @@ def add_route_to_plc(
     route_name=None,
     added_net_id=None,
 ):
-    # type: (AmsAddr, str, str, str, str, str, AmsAddr) -> None
+    # type: (str, str, str, str, str, str, str) -> bool
```
Comment (stlehmann) [inline pyads/ads.py:364]: That's right, the type hints were wrong, here

File: tests/test_plc_route.py [hunk 2]
```diff
@@ -53,13 +53,13 @@ def plc_route_reciever(self):
 
             data = data[4:]  # Remove protocol bytes
 
-            len_sending_host = struct.unpack("<H", data[:2])[0]  # Length of host name
+            len_route_name = struct.unpack("<H", data[:2])[0]  # Length of PLC password
```
Comment (stlehmann) [inline tests/test_plc_route.py:56]: route name or plc password? There is a contradiction here between the comment and the variable name

**Comment by stlehmann** [comment]
Could you please rebase your PR on the current master branch and add a changelog entry?

**Comment by chrisbeardy** [comment]
I'll take a look at this shortly.

**Comment by chrisbeardy** [comment]
one sec, accident...

**Comment by chrisbeardy** [comment]
sorry, when I squashed, I forgot I had already pushed to a remote, so it caused a merge error, I handled this incorrectly and accidentally pushed all the old commits into my remote. So had to correct this, hence the new PR.

**Pull request closed without merging**